h2o-algos/src/main/java/hex/ensemble/StackedEnsemble.java

```diff
@@ -92,6 +100,25 @@ public boolean haveMojo() {
     return true;
   }
 
+  @Override
+  public int nclasses() {
+    DistributionFamily distribution;
+    if (_parms._metalearner_parameters != null) {
+      if (_parms._metalearner_parameters instanceof GLMModel.GLMParameters)
+        distribution = familyToDistribution(((GLMModel.GLMParameters) _parms._metalearner_parameters)._family);
+      else
+        distribution = _parms._metalearner_parameters._distribution;
+
+      if (multinomial.equals(distribution) || ordinal.equals(distribution) || AUTO.equals(distribution))
+        return _nclass;
+      if (bernoulli.equals(distribution) || quasibinomial.equals(distribution)
+              || fractionalbinomial.equals(distribution))
+        return 2;
+      return 1;
+    }
+    return super.nclasses();
+  }
+
```

Comment:
Fix problem with quasibinomial/fractionalbinomial distributions in SE - they need to be interpreted as classification not regression (which is the current state).